js-miniapp-sdk/src/types/responseTypes/AdResponse.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Response from native sdk for ads
+ * miniAppId is a GUID string
+ * adUnitId is string eg ca-app-pub-7941117952683310/1302674583
+ * startTime is ad's load start Time
+ * endTime is the ad's unload time
+ * provider is the active ad provider network selected
+ */
+export interface AdResponse {
+  miniAppId?: string;
+  adUnitId?: string;
```

Comment:
Removed!

Reply:
Ad Unit ID needs to works with conjuction with App Id,
I considered this before so that the return result can be fed to an api for later usage for Ad revenue audit!